noaa/challenges/default.json

```diff
@@ -356,205 +358,4 @@
           "target-throughput": 1
         }
       ]
-    },
-    {
-      "name": "aggs",
```

Comment:
we should keep this challenge, at least for transitional purposes. it can just unconditionally `include 'aggs-tasks.json'`